StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Elements/InlineSignup/LinkInlineSignupView.swift

```diff
@@ -238,19 +267,13 @@ final class LinkInlineSignupView: UIView {
     }
 
     private func updateAppearance() {
-        backgroundColor = viewModel.configuration.appearance.colors.background
-
-        var cornerRadius = viewModel.configuration.appearance.cornerRadius
-        if !viewModel.bordered {
-            // If we're not bordered, the content is right at the border of the view.
-            // Remove corner radius so that we don't cut off anything.
-            cornerRadius = 0
-        }
-        layer.cornerRadius = cornerRadius
+        backgroundColor = viewModel.containerBackground
+        layer.cornerRadius = viewModel.containerCornerRadius
 
         // If the borders are hidden give Link a default 1.0 border that contrasts with the background color
-        if viewModel.configuration.appearance.borderWidth == 0.0 ||
-            viewModel.configuration.appearance.colors.componentBorder.rgba.alpha == 0.0 {
+        let hasInvisibleBorder = viewModel.configuration.appearance.borderWidth == 0.0 ||
+            viewModel.configuration.appearance.colors.componentBorder.rgba.alpha == 0.0
+        if viewModel.bordered && hasInvisibleBorder {
```

Comment:
The `hasInvisibleBorder` workaround no longer works due to [this change](https://github.com/stripe/stripe-ios/blob/master/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift#L38). We un-hide the border if needed.